frontend/src/uxbox/util/dom.cljs

```diff
@@ -162,6 +162,19 @@
   {:width (.-clientWidth ^js node)
    :height (.-clientHeight ^js node)})
 
+(defn get-bounding-rect
+  [node]
+  (let [rect (.getBoundingClientRect ^js node)]
```

Comment:
the same here, `^js` meta tag is needed  on all interop calls.

Reply:
Changed. Not sure if well done. Other functions in this module do not always have the tag.

Reply:
Many interop calls are already covered by builtin externs. but is a good practice tagging all the interpor calls for avoid future headaches with the production builds.